README.md

```diff
@@ -139,6 +139,77 @@ tox -e kubeflow-remote
 tox -e kubeflow-local
 ```
 
+### Run behind proxy
+
+#### Running using Notebook
+
+##### Prerequistes
+
+Edit the PodDefault `tests/proxy-poddefault.yaml` to replace the placeholders for:
+
+* `http_proxy` and `https_proxy` - The address and port of your proxy server, format should be `<proxy_address>:<proxy_port>`
+* `no_proxy` - A comma separated list of items that should not be proxied. It is recommended to include the following:
+
+`<cluster cidr>,<service cluster ip range>,127.0.0.1,localhost,<nodes internal ip(s)>/24,<cluster hostname>,.svc,.local`
+
+where,
+
+  * `<cluster cidr>`: you can get this value by running:
+
+    ```
+    cat /var/snap/microk8s/current/args/kube-proxy | grep cluster-cidr
+    ```
+
+  * `<service cluster ip range>`: you can get this value by running:
+
+    ```
+    cat /var/snap/microk8s/current/args/kube-apiserver | grep service-cluster-ip-range
+    ```
+   
+  * `<nodes internal ip(s)>`: the Internal IP of the nodes where your cluster is running, you can get this value by running:
+
+    ```
+    microk8s kubectl get nodes -o wide
+    ```
+    It is the `INTERNAL-IP` value
+
+  * `<hostname>`: the name of your host on which the cluster is deployed, you can use the `hostname` command to get it
+
+  * `localhost` and `127.0.0.1` are recommended to avoid proxying requests to `localhost`
+
+
+To run the tests behind proxy using Notebook:
+1. Login to the Dashboard and Create a Profile
+2. Apply the PodDefault to your Profile's namespace, make sure you already followed the Prerequisites
+   section to modify the PodDefault. Apply it with:
+   ```
+   kubectl apply -f ./tests/proxy-poddefault.yaml -n <your_namespace>
+   ```
+3. Create a Notebook and from the `Advanced Options > Configurations` select `Add proxy settings`,
+   then click `Launch` to start the Notebook.
+   Wait for the Notebook to be Ready, then Connect to it.
+4. From inside the Notebook, start a new terminal session and clone this repo:
+
+   ```bash
+   git clone https://github.com/canonical/charmed-kubeflow-uats.git
+   ```
+   Open the `charmed-kubeflow-uats/tests` directory and for each `.ipynb` test file there, open it
+   and run the Notebook.
+   
+   Currently, the following tests are supported to run behind proxy:
+   * katib
+   * kserve
+   * kfp_v2
+   * training (except TFJob due to https://github.com/canonical/training-operator/issues/182)
```

Comment:
```suggestion
   * training
```
the tfjob issue is now fixed

Reply:
I will send a PR against the dev branch for removing this note.